.github/workflows/fetch-news.yml

```
@@ -0,0 +1,33 @@
+name: Fetch Discourse News
```

Similarly to the file comment:


```suggestion
name: Update Discourse data
```